app/scripts/modules/amazon/src/loadBalancer/configure/classic/createClassicLoadBalancer.controller.ts

```diff
@@ -149,11 +150,16 @@ export class CreateClassicLoadBalancerCtrl extends CreateAmazonLoadBalancerCtrl
     }
     loadBalancer.healthCheck = healthCheck;
   }
+
+  public validateTimeout(value: number): boolean {
+    return value < this.loadBalancerCommand.healthInterval;
+  }
 }
 
 export const AWS_CREATE_CLASSIC_LOAD_BALANCER_CTRL = 'spinnaker.amazon.loadBalancer.classic.create.controller';
 module(AWS_CREATE_CLASSIC_LOAD_BALANCER_CTRL, [
   require('@uirouter/angularjs').default,
+  'ui.validate',
```

Comment:
Yea, it doesn't export the module so require doesn't work.  this lib is...old...  But, good thing is if @anotherchrisberry's suggestion works, we don't need it.